### PR TITLE
Allow dispatch to reorder timeslots on Store settings

### PR DESCRIPTION
**Pull request by r0xsh**
Time slots are now managed via a self-contained React component. 

time_slots_react

### CODE REVIEW

File: src/Entity/StoreTimeSlot.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ *     "patch"={
+ *       "method"="PATCH",
+ *     },
+ *     "put"={
+ *       "method"="PUT",
+ *     },
+ *   }
+ * )
+ * @ApiFilter(SearchFilter::class, properties={"store"})
+ */
```
Comment (alexsegura) [inline src/Entity/StoreTimeSlot.php:24]: Not sure it is actually useful to expose via API

File: assets/react/controllers/Store/TimeSlot.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+}
+
+const Row = ({ row, index, setSelectedTS, selectedTS }) => (
+  <Draggable draggableId={index.toString()} index={index}>
+    {(provided) => (
+      <p key={index} ref={provided.innerRef} {...provided.draggableProps}>
+        <i {...provided.dragHandleProps} className="fa fa-bars mr-1"></i>
+        <i
+          className="fa fa-minus-circle"
+          onClick={() => setSelectedTS(removeValue(selectedTS, row.value))}
```
Comment (alexsegura) [inline assets/react/controllers/Store/TimeSlot.jsx:41]: This should trigger an HTTP request to update the list as well.

**Review by alexsegura — COMMENTED** [review]
Translations are missing.

**Comment by r0xsh** [comment]
Tested and everything works as expected, ty @alexsegura for your help :)

**Pull request merged**